packages/console/src/consts/plan-quotas.ts

```diff
@@ -9,6 +10,8 @@ import {
   type SubscriptionPlanQuota,
 } from '@/types/subscriptions';
 
+import { isDevFeaturesEnabled as isDevelopmentFeaturesEnabled } from './env';
```

Comment:
nit: no need to rename?

Reply:
It's changed by the lint rule, will update the rule in another PR later.